internal/p2p/router.go

```diff
@@ -445,8 +445,8 @@ func (r *Router) dialSleep(ctx context.Context) {
 		// limits peers for dialing more than once every 10ms,
 		// so these numbers are safe.
 		const (
-			maxDialerInterval = 500 // ms
-			minDialerInterval = 100 // ms
+			maxDialerInterval = 30 // ms
+			minDialerInterval = 10 // ms
```

Comment:
I think it would be viable to take dialsleep out entirely. it's existance is really "we're dialing too fast to make progress," but I think that's likely to not be a thing that happens (or more likely if it does happen, we'd be able to see in the ratio of the connection success/failure metrics, and is far more likely to impact the e2e cases than any production case,)

Reply:
If dial sleep were taken out, would we eventually block on the `addresses` channel. i.e. `DialNext` could only add a new address when one of the dialing routines picked the previous address off the channel?

Reply:
That would be no different than the current behavior it just might happen a bit sooner.